backend-models/service-tests/accessedservice-tests.txt

```diff
@@ -0,0 +1,129 @@
+
+accessedservice.php
+
+-------------------
+
+********
+* pre-req
+********
+
+(checklogin() is TRUE && (hasAccess($userid, $cid, 'w') || isSuperUser($userid,) is TRUE))
+
+--------------------
+
+Test 1: try to update a created user by changing the username
+Row: 62 - 80 and 220 - 238
+Send {
+   $opt = "ADDUSER"
```

Comment:
$opt = "ADDUSER"   should it not be "UPDATE"?
![image](https://user-images.githubusercontent.com/102600391/234264272-fa2f3bba-003a-4ab5-8575-c817933e70da.png)